src/libs/antares/study/area/links.cpp

```diff
@@ -217,14 +217,23 @@ bool AreaLink::linkLoadTimeSeries_for_version_820_and_later(const AnyString& fol
     String filename;
     bool success = true;
 
+    bool enabledModeIsChanged = false;
```

Comment:
Class Matrix is so messy that we're compelled to do things like that !
We should clean this class some day

Reply:
I completely agree. Matrix + JIT is a mess.